lib/resultsCapturer.js

```diff
@@ -245,8 +253,8 @@ function commonStepFinished({
         scenarioFailed = true;
         stepResults.assertions.push({
           passed: false,
-          errorMsg: exception.message,
-          stackTrace: exception.stack
+          errorMsg: exception ? exception.message : 'No Exception message found',
+          stackTrace: exception ? exception.stack : undefined
```

Comment:
I think `exception?.stack` will be better.

Reply:
I agree!